composer.json

```diff
@@ -57,6 +58,12 @@
     ],
     "phpcbf": [
       "phpcbf -p"
+    ],
+    "makepot-audit": [
+      "wp i18n make-pot . --exclude=\".github,.wordpress-org,bin,sample-data,node_modules,tests\""
```

Comment:
It doesn't exclude the `vendor` directory anymore, since we may include some strings, like for the `jetpack-autoloader` package.
It also should run in the `vendor` directory for our packages, they also include some translatable strings.

Reply:
Yeah, I noticed it as well as I was playing around with it today.

Reply:
Checked now and even though `vendor` is not excluded, e.g. the string "Not all plugins have loaded yet but we requested the class %s" from jetpack domain has not been extracted. Maybe it needs to first get processed by wp-textdomain? Not sure. But that's a minor issue.

Reply:
@peterfabian we are not including all text domains on this script, so will only include what have already `woocommerce`.